compiler/src/dotty/tools/MainGenericRunner.scala

```diff
@@ -199,11 +200,13 @@ object MainGenericRunner {
             Option.when(Jar.isJarOrZip(dotty.tools.io.Path(entryPath)))(Jar(entryPath).mainClass).flatten
           }.isDefined
         }
-        targetToRun match
+        val newSettings =
+          targetToRun match
```

Comment:
This indentation was also to facilitate `println` and can be reverted. It's kind of cool that the indentation works as advertised.